### ISSUE TITLE
max value for rev_id in the revision table

### ISSUE THREAD

**Issue by dmcmilla** [UB-Mannheim/RaiseWikibase#10]
The method to get the next rev_id doesn't always match the next value that will be inserted in the revision table.  The revision table uses a sequence to insert the next row.  In the case where a record was deleted, max +1 would not equal the next sequence # of the rev_id.  It's probably not an issue on a fresh system or one where no deletes have been performed.

**Comment by renat-kaufmann** [comment]
Right! The main focus was indeed on a fresh Wikibase instance, without deleted entities in mind.
